actix-limitation/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "actix-limitation"
+version = "0.1.4"
+authors = ["errmac-v"]
+description = "A rate limiter using a fixed window counter for arbitrary keys, backed by Redis for actix-web."
+keywords = ["actix-web", "rate-api", "rate-limit", "limitation"]
+categories = ["asynchronous", "web-programming"]
+repository = "https://github.com/actix/actix-extras.git"
+license = "MIT"
```

Comment:
@0xmad our crates are dual licensed as MIT or Apache 2.0. I need you to confirm this is okay to change.

Reply:
Yes, feel free to update it.